includes/collection/class-users.php

```diff
@@ -57,12 +57,13 @@ public static function get_by_id( $user_id ) {
 			);
 		}
 
-		if ( self::BLOG_USER_ID === $user_id ) {
-			return new Blog();
-		} elseif ( self::APPLICATION_USER_ID === $user_id ) {
-			return new Application();
-		} elseif ( $user_id > 0 ) {
-			return User::from_wp_user( $user_id );
+		switch ($user_id) {
+			case self::BLOG_USER_ID:
+				return new Blog();
+			case self::APPLICATION_USER_ID:
+				return new Application();
+			default:
+				return User::from_wp_user($user_id);
```

Comment:
WordPress Coding Style:

```suggestion
				return User::from_wp_user( $user_id );
```